src/lib/connections/wireguard

```diff
@@ -0,0 +1,30 @@
+# Contributed by: Thibaut Sautereau (thithib) <thibaut at sautereau dot fr>
+
+. "$SUBR_DIR/ip"
+
+wireguard_up() {
+    if is_interface "$Interface"; then
+        report_error "Interface '$Interface' already exists"
+        return 1
+    fi
+
+    interface_add wireguard "$Interface"
+    if ! is_interface "$Interface"; then
+        # This check should be removed once WireGuard support lands in the Arch
+        # Linux kernel. Alternatively, we could add 'WIREGUARD-MODULE' to the
+        # PKGBUILD optdepends?
```

Comment:
Believe it or not, `netctl` is not specific to Arch Linux. I think the above comment can simply be dropped altogether.